lib/blockchain/common.js

```diff
@@ -99,7 +105,11 @@ function fetchAndInstall (coinRec) {
   es(`wget -q ${url}`)
   es(`tar -xzf ${downloadFile}`)
 
-  if (_.isEmpty(binaries.files)) {
+  if (binaries.build) {
+    es(`${__dirname}/${binaries.build.script} \`realpath ${binaries.build.folder}\``)
+  }
+
+  if (_.isEmpty(binaries.files) && binDir) {
```

Comment:
Why is there `&& binDir` now? Unless I am mistaken, this does not affect cardano at all but might affect other currencies?